crates/metal-codegen-llvm/src/linking.rs

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+
+use std::{env, fs, process::Command};
+
+fn prefix_dir(s: &str) -> String {
+    if cfg!(target_os = "windows") {
+        "\\".to_string() + s
+    } else {
+        "/".to_string() + s
+    }
+}
+
+pub fn get_lld_dir() -> String {
+    let mut output = if let Ok(p) = env::var("LLVM_SYS_191_PREFIX") {
+        p + &prefix_dir("bin")
+    } else if let Ok(p) = env::var("METAL_LLD_DIR") {
+        p
+    } else {
+        let output = Command::new("llvm-config").arg("--bindir").output();
+
+        if let Ok(output) = output {
+            String::from_utf8_lossy_owned(output.stdout).replace("\n", "")
```

Comment:
Maybe strip instead?

Reply:
Just removed the replace. Not sure what it was there for previously.